project/projectClient.ts

```diff
@@ -1,6 +1,8 @@
 import datasources from "./datasources.json";
 import metrics from "./metrics.json";
+import precalc from "./precalc.json";
 import objectives from "./objectives.json";
+import geographies from "./geographies.json";
```

Comment:
precalc and geography files are both now imported by projectClient.  So they are required for each project.